opts/opts.go

```diff
@@ -179,6 +179,24 @@ func ValidateLink(val string) (string, error) {
 	return val, nil
 }
 
+// ValidateCpusetCpus validates that the specified string has a valid
+// range format
+func ValidateCpusetCpus(val string) (string, error) {
+	if _, err := parsers.ParseAvailableIntString(val); err != nil {
+		return val, err
+	}
+	return val, nil
+}
+
+// ValidateCpusetMems validates that the specified string has a valid
+// range format
+func ValidateCpusetMems(val string) (string, error) {
```

Comment:
This looks exactly the same as `ValidateCpusetCpus`, how about 1 function called `ValidateCPUSetList`